src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When did World Wide Web Consortium first drafted standards for developing accessible websites?",
+    Answer: "1999",
+    Distractor1: "2001",
+    Distractor2: "1990",
+    Distractor3:
+      "1989",
+    Explanation:
+      "The World Wide Web Consortium first drafted standards for developing accessible websites back in 1999.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+  },
+  {
+    Question:
+      "What legal requirement has brought web accessibility to the forefront in recent years?",
+    Answer:"The Americans with Disabilities Act",
+    
+    Distractor1: "The Internet Freedom Act",
+    Distractor2: "The Web Accessibility Act",
+    Distractor3:
+      "The Digital Inclusion Law",
+    Explanation:
+"The Americans with Disabilities Act (ADA) has brought web accessibility to the forefront by requiring public spaces and commercial facilities to provide equal access to individuals with disabilities, which now includes web-based activities due to legal rulings.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+  },
+  {
+    Question:
+      "Besides people with disabilities, who else benefits from following Web accessibility guidelines?",
+    Answer: "People with low literacy and search engines",
+    Distractor1: "Only screen reader users",
+    Distractor2: "People with low literacy and search engines",
+    Distractor3:
+      "Only website developers",
+    Explanation:
+      "Web accessibility guidelines benefit people with low literacy by using clear language and improve search engine indexing, making content more accessible and discoverable.",
```

Comment:
this question is a little misleading because technically everyone benefits from good accessibility
let's go with this rewording instead

 ```suggestion
    Question: "Which of the following is NOT a benefit of web accessibility?",
    Answer: "Limited Audience Reach",
    Distractor1: "Higher Search Engine Rankings",
    Distractor2: "Inclusivity",
    Distractor3:
      "Improved User Experience",
    Explanation:
      "Making websites accessible broadens the audience reach rather than limiting it, ensuring that everyone, regardless of their abilities, can access the content online.",
```